native/src/modules/layout/__tests__/HeaderContainer.spec.js

```diff
@@ -132,41 +163,28 @@ describe('HeaderContainer', () => {
         }
       }
     }
-
-    const result = TestRenderer.create(
-      <Provider store={store}>
-        {/* $FlowFixMe not all props passed */}
-        <HeaderContainer {...ownProps}/>
-      </Provider>
-    )
-
-    const header = result.root.findByType(Header)
-
-    expect(header.props).toEqual(
-      expect.objectContaining({ shareUrl: cityContentUrl({ cityCode: city.code, languageCode: language._code, route: EVENTS_ROUTE, path: null }) })
-    )
+    const result = render(ownProps)
+    const expectedShareUrl = cityContentUrl({ cityCode: city.code, languageCode: language._code, route: EVENTS_ROUTE, path: null })
```

Comment:
```suggestion
    const expectedShareUrl = cityContentUrl({ cityCode: city.code, languageCode: language.code, route: EVENTS_ROUTE, path: null })
```
Same for other occurrances

Reply:
done